README.md

```diff
@@ -8,6 +8,13 @@ See: http://aem.live/docs/sidekick-generate-variations
 
 ### Initial Setup
 
+- Check that you have access to [App Builder](https://developer.adobe.com/app-builder/) and [Developer Console](https://developer.adobe.com/console) (log in with the **Sites Internal** profile)
+    - Request access from an IT admin
+- Install dependencies: `npm install`
+- Select an organization, project, and workspace:
+    - `aio console org select`
+    - `aio console project select`
+    - `aio console workspace select`
```

Comment:
@audrey-kho Can you investigate App Builder Quickstarts by watching this video and see if selecting org, project, and workspace can be simplified?
Link: https://www.youtube.com/live/5h5_UcxUYhU?si=7zCNq_0Rrc9IlrXw&t=1294